images/libvirt/Dockerfile.ci

```diff
@@ -5,6 +5,8 @@
 FROM registry.ci.openshift.org/ocp/4.16:installer-terraform-providers as providers
 
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16 AS builder
+# FIPS support is offered via the baremetal-installer image
+ENV GO_COMPLIANCE_EXCLUDE=".*"
```

Comment:
libvirt image has never been static
(we are likely to delete this soon anyway)

Reply:
Ok. I've dropped the static change from the libvirt image. We can deal with it in a separate PR.